tests/vmi_configuration_test.go

```diff
@@ -407,6 +407,44 @@ var _ = Describe("Configurations", func() {
 
 		})
 
+		Context("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:component]Support memory over commitment test", func() {
+			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+			vmi.Spec.Domain.Resources.Requests[kubev1.ResourceMemory] = resource.MustParse("1G")
```

Comment:
Is the 1G hard requirement? The tests would work for much less memory. Generally our tests should require only resources that are necessary. Making the memory smaller will improves the overall CI run.

Reply:
Not a hard requirement, it was mentioned in the Polarion test case but shouldn't matter if it's something like 128M, changed now